shared/signup/phone-number/index.tsx

```diff
@@ -15,10 +15,9 @@ export type Props = {
 const EnterPhoneNumber = (props: Props) => {
   const [phoneNumber, onChangePhoneNumber] = React.useState('')
   const [valid, onChangeValidity] = React.useState(false)
-  // const [allowSearch, onChangeAllowSearch] = React.useState(false)
+  const [allowSearch, onChangeAllowSearch] = React.useState(true)
```

Comment:
Did we reverse the earlier decision to disable toggling this during signup?

Reply:
I wasn't aware of that decision. I any case, the default during signup should be to allow search. If we don't show the checkbox, doesn't the `makePhoneRow.searchable` above need to stay true?

Reply:
The previous code was passing `true` to the callback in lieu of showing the checkbox

Reply:
Ok I'll restore it